dev/preview/previewctl/cmd/get_name.go

```diff
@@ -8,16 +8,17 @@ import (
 	"fmt"
 
 	"github.com/gitpod-io/gitpod/previewctl/pkg/preview"
+	"github.com/go-kit/log"
```

Comment:
All code in Gitpod uses `logrus` - we even tie `klog` (used by Kubernetes components) back to logrus.

What's the motivation to go astray? I.e. which requirements warrant the use of a different library?

Reply:
None really, just wanted to move away from the previous 🙂

I'll update the PR to logrus

Reply:
best use `common-go/log`, but using logrus directly is fine, too.

Reply:
I believe adopting a big lib just for one of the features may be too much. 

I've only switched to logrus for now, but if someday in the future we need tracing/metrics/pprof, then common-go sounds like a good idea 🙂